app/models/event/accessioning_event.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class Event::AccessioningEvent < Event
+  def self.created_accession_number!(eventable, accession_type, user)
```

Comment:
might be worth adding some comments here to explain when an update event is made as opposed to a created event.